controllers/login.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const router = express.Router();
+const { User } = require("../models/schema");
+const { createToken } = require("../models/token");
+
+router.post("/login", async (req, res) => {
+  const body = req.body;
+  const { email, password } = body;
+  const user = await User.findOne({ email });
+  if (!user) {
+    return res.json({
+      status: "error",
+      code: 400,
+      message: "User whit this email does not exist",
+    });
+  }
+  if (!user || !user.validPassword(password)) {
+    return res.json({
+      status: "error",
+      code: 401,
+      message: "Password is wrong",
+    });
+  }
+  const token = createToken(user._id);
```

Comment:
Tu worzysz token ale musisz jeszcze zrobić update bazy danych. Czyli przypisać ten token do użytkownika, podobnie jak to robisz przy tworzeniu nowego uzytkownika. :)